### PR TITLE
added more fields

### PR DESCRIPTION
**Pull request by Bibiko**

### CODE REVIEW

File: cobl2/__init__.py [hunk 1]
```diff
@@ -66,4 +67,12 @@ def main(global_config, **settings):
     config.add_route('test', '/test')
     config.registry.registerUtility(link_attrs, ILinkAttrs)
     config.registry.registerUtility(CoblMapMarker(), IMapMarker)
+    menuitems = [
+            'languages',
+            'parameters',
+            ('cognatesets', dict(label='Cognate Sets')),
```
Comment (xrotwang) [inline cobl2/__init__.py]: Is there a reason to not use the translations solution here? Presumably, "Cognatesets" appears more often in the user interface of the app?

File: cobl2/templates/cognateset/detail_html.mako [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <%inherit file="../${context.get('request').registry.settings.get('clld.app_template', 'app.mako')}"/>
 <%namespace name="util" file="../util.mako"/>
 <%! active_menu_item = "cognatesets" %>
-<%block name="title">${_('Cognateset')} ${ctx.name}</%block>
+<%block name="title">${_('Cognate Set')} ${ctx.name}</%block>
```
Comment (xrotwang) [inline cobl2/templates/cognateset/detail_html.mako]: Did translation not work? Putting the exact string you want in the UI into `_('...')` can only make things worse, btw.

File: cobl2/templates/cognateset/index_html.mako [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<%inherit file="../${context.get('request').registry.settings.get('clld.app_template', 'app.mako')}"/>
+<%namespace name="util" file="../util.mako"/>
+<%! active_menu_item = "cognatesets" %>
+<%block name="title">${_('Cognate Sets')}</%block>
```
Comment (xrotwang) [inline cobl2/templates/cognateset/index_html.mako]: Overwriting a complete template just to tune the title seems overkill.

File: cobl2/__init__.py (hunk 1 above)
Comment (Bibiko) [reply cobl2/__init__.py]: how can be queried the translated name here?

**Pull request merged**